samples/spring-eventsourced-shopping-cart/src/main/resources/application.conf

```diff
@@ -0,0 +1,4 @@
+
+# //tag::snapshot-every[]
+kalix.event-sourced-entity.snapshot-every = 100
```

Comment:
Which bring us to the fact that we are not yet supporting entity options programmatically. Users can define their options in code as well. 

I will create a ticket for it.